doc/WhatsNew.rst

```diff
@@ -27,6 +27,14 @@ Ver 3.3.0 (unreleased)
 - Changes to 'histogram' and 'stddev' autocuts algorithms:
   - choice of sampling by grid now; useful for mosaics and collages
   - for previous parameter of usecrop=True, use sample=crop
+- Moved loading of FITS HDUs from AstroImage to io_fits module,
+  encapsulating the details of this file format into the module
+  responsible for loading those files
```

Comment:
Maybe this will fix `doc/WhatsNew.rst:36: ERROR: Unexpected indentation`?
```suggestion
  responsible for loading those files:
```

Reply:
Looked at an example further down in the file.  Looks like I needed the colon and perhaps a blank line for RST to recognize the sublist.